src/handlers/guild.ts

```diff
@@ -44,7 +44,7 @@ import { urlToBase64 } from "../utils/utils.ts";
 
 /** Create a new guild. Returns a guild object on success. Fires a Guild Create Gateway event. This endpoint can be used only by bots in less than 10 guilds. */
 export function createServer(options: CreateServerOptions) {
-  return RequestManager.post(endpoints.GUILDS, options);
+  return RequestManager.post(endpoints.GUILDS, options) as Guild;
```

Comment:
It doesn't return guild it returns the create guild payload. Sidenote though i think i did this already in the next branch if im not mistaken

Reply:
Oh, alright! I will close this.